lib/cmd/fh3/keys/ssh/add.js

```diff
@@ -0,0 +1,52 @@
+/* globals i18n */
+var common = require('../../../../common.js');
+var fhreq = require("../../../../utils/request");
+var fs = require('fs');
+var fhc = require("../../../../fhc");
+
+module.exports = {
+  'desc' : i18n._('Add a SSH Key Management'),
+  'examples' :
+    [{
+      cmd : 'fhc keys ssh add --label=<label> --file=<file>',
+      desc : i18n._("Add new SSH Key Management which is into the path <file> with the <label>")
+    }],
+  'demand' : ['label','file'],
+  'alias' : {
+    'label': 'l',
+    'json': 'j',
+    0 : 'label',
+    1 : 'file'
+  },
+  'describe' : {
+    'label' : i18n._("Label of the ssh key that you want add."),
+    'file' : i18n._("Path with file name of the ssh key that you want add."),
+    'json' : i18n._("Output in json format")
+  },
+  'customCmd': function(params, cb) {
+    if (!fs.existsSync(params.file)) {
+      return cb(i18n._("File doesn't exist: ") + params.file);
+    }
+
+    fs.readFile(params.file, function(err, key) {
+      if (err) {
+        return cb(err);
+      }
+
+      var url = "box/srv/1.1/ide/" + fhc.curTarget + "/user/addKey";
+      var payload = {
+        label: params.label,
+        key: key.toString()
+      };
+      common.doApiCall(fhreq.getFeedHenryUrl(), url, payload,  i18n._("Error adding Key: "), function(err,data) {
```

Comment:
Do we really need to use `i18n` for error messages too? I think we don't usually do that in fhc (but i may be wrong).